ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh

```diff
@@ -78,12 +78,24 @@ pushd "${BASE_DIR}" || exit
 mkdir logs
 pushd logs || exit
 
+# check if we have any kuttl namespace
+KUTTL_NS=$(oc get project -o name | grep kuttl | cut -d '/' -f 2)
+
 # api-resources - OpenStack
 CMD=$(gen_resource_yaml_debug_cmd "${NS_SERVICES}" "api-resources")
 oc -n "${NS_SERVICES}" api-resources | grep 'openstack' | awk '{print $1}' | xargs -n1 -I {} sh -c "${CMD[@]}"
 # RabbitMQCluster
 oc -n "${NS_SERVICES}" get -o yaml RabbitMQCluster > RabbitMQCluster.yaml
 
+if [ -n "${KUTTL_NS}" ]; then
```

Comment:
I was checking logs and I see api-resources-kuttl/ dir[1], interesting only `api-resources-kuttl` dir was created not pods_kuttl and errors_kuttl. 

I wonder why `api-resources-kuttl` dir was created if kuttl test was passed and cleanup was run.

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41021/rehearse-41021-pull-ci-openstack-k8s-operators-heat-operator-main-heat-operator-build-deploy-kuttl/1678786216360677376/artifacts/heat-operator-build-deploy-kuttl/openstack-k8s-operators-gather/artifacts/api-resources-kuttl/

Reply:
It's created because it calls `oc api-resources` which returns a list of the crds installed, regardless of there is actually any CR created (in the case you pointed they are all deleted, which is why all the describe files are empty). Even when we have cleaned up and there are not pods, CRs or any other resource, the crds are still in the cluster. We could add some logic to check whether there is any CR and avoid the creation otherwise, but with the move to `must-gather` I'm not sure it's worth it.

Reply:
Spoke with Joan over irc.

I was under the assumption that we delete Kuttl namespace after the tests so I was expecting we would not enter if condition at all.

But that is not the case right now, we don't delete kuttl namespace currently.